src/components/Search/SearchPageHeader.tsx

```diff
@@ -222,7 +222,7 @@ function SearchPageHeader({
                     shouldAlwaysShowDropdownMenu
                     pressOnEnter
                     buttonSize={CONST.DROPDOWN_BUTTON_SIZE.MEDIUM}
-                    customText={translate('workspace.common.selected', {selectedNumber: selectedTransactionsKeys.length})}
+                    customText={translate('workspace.common.selected', undefined, selectedTransactionsKeys.length)}
```

Comment:
Having undefined just because you have to provide the third argument isn't ideal. I'm okay with leaving it like this but is there another way?

Reply:
> It will force every parameter block to use Object structure. Even though there is only one parameter.

Why is this bad? This was how it worked till it changed [here](https://github.com/Expensify/App/pull/29742/files#r1459284554). I have no idea why that change was made.

> It will require more extensive changes as it will require to change many translation parameter structures

What do you mean? Can you list some of them here?

>  It will remove the optional parameter typing (which will require us to pass an empty structure instead of nothing).

Why? I don't see why we could not still make the parameters param be optional.

Reply:
> How will you detect if the type has optional members?

What do you mean?

Reply:
@shubham1206agra Any update here on moving ahead?

Reply:
>  I will try to do this in another follow-up PR.

Okay!
@iwiznia What are your thoughts on the other follow-up PR?

Reply:
> We have been waiting for this feature for over 6 months, we can wait longer and make it right, especially since if we change it later we will need to update all callers of the method using the count param.

Fair enough, @shubham1206agra We need to make the changes in this PR only, so please share any TypeScript errors you're encountering when adding the count to the phrase parameters. We can work together to resolve them and also get a second opinion from the TypeScript experts.

Reply:
@iwiznia @shubham1206agra I'm working on adding the count to the phrase parameters. Currently, I'm encountering 3-4 TypeScript errors and still need to test a few things. I'll push the code for review over the weekend.

Reply:
@shubham1206agra I've raised a draft PR https://github.com/Expensify/App/pull/47963 where I've attempted to pass the count parameter within the phrase parameters. However, I'm encountering a TypeScript error that I still need to resolve. I'd appreciate your thoughts on the PR and any feedback you might have on the changes.